WordPressKit/IPLocationRemote.swift

```diff
@@ -16,10 +16,7 @@ public final class IPLocationRemote {
     /// Fetches the country code from the device ip.
     ///
     public func fetchIPCountryCode(completion: @escaping (Result<String, Error>) -> Void) {
-        let path = WordPressComOAuthClient.WordPressComOAuthDefaultApiBaseUrl + "/geo/"
-        guard let url = URL(string: path) else {
-            return completion(.failure(IPLocationError.malformedURL))
```

Comment:
I did a quick `grep` in the codebase and I think this removal also results in the `malformedURL` error becoming unused.

Since we're aiming to ship a new major version, might as well remove that, too?